dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/zk/ZookeeperOperator.java

```diff
@@ -136,17 +138,18 @@ public String get(final String key) {
     }
 
     public List<String> getChildrenKeys(final String key) {
-        List<String> values;
         try {
-            values = zkClient.getChildren().forPath(key);
-            return values;
+            return zkClient.getChildren().forPath(key);
         } catch (InterruptedException ex) {
             logger.error("getChildrenKeys key : {} InterruptedException", key);
             throw new IllegalStateException(ex);
         } catch (Exception ex) {
-            logger.error("getChildrenKeys key : {}", key, ex);
```

Comment:
It may be better to judge whether it is NoNodeException by the exception type than by msg